domain/api/atrium-domain-api-jvm/src/main/kotlin/ch/tutteli/atrium/domain/creating/PathAssertions.kt

```diff
@@ -17,5 +17,6 @@ val pathAssertions by lazy { loadSingleService(PathAssertions::class) }
  * which an implementation of the domain of Atrium has to provide.
  */
 interface PathAssertions {
-    fun <T: Path> exists(assertionContainer: Expect<T>): Assertion
+    fun <T : Path> exists(assertionContainer: Expect<T>): Assertion
+    fun <T : Path> existsNot(subjectProvider: Expect<T>): Assertion
```

Comment:
also use assertionContainer instead of subjectProvider, same same at other places